### PR TITLE
Update ipc-main.md

### PR DESCRIPTION
**Pull request by Daniel-Zury15**
Sorry

Correction: 

```js title='preload.js'
// In preload.
const { contextBridge, ipcRenderer } = require('electron')

```

### CODE REVIEW

**Review by codebytere — COMMENTED** [review]
hi @Daniel-Zury15 - can you please explain what you're trying to accomplish here? you've provided no PR description and the code changes do not make clear what you are trying to fix or add. We can proceed reviewing this PR once you have done so.

File: docs/api/ipc-main.md [hunk 1]
```diff
@@ -24,33 +24,52 @@ process, see [webContents.send][web-contents-send] for more information.
...
-  event.reply('asynchronous-reply', 'pong')
-})
 
+// A synchronous message.
 ipcMain.on('synchronous-message', (event, arg) => {
   console.log(arg) // prints "ping"
   event.returnValue = 'pong'
 })
+
+// A asynchronous message.
```
Comment (miniak) [inline docs/api/ipc-main.md:37]: An asynchronous

**Review by erickzhao — COMMENTED** [review]
This will likely conflict with https://github.com/electron/electron/pull/32059. Would prefer closing this and working on getting the IPC guide merged, personally.

**Comment by jkleinsc** [comment]
- Closing in favor of #32059

**Pull request closed without merging**